### PR TITLE
excmds.ts: make a lot of functions return promises

### PR DESCRIPTION
**Pull request by glacambre**
It's important to have async functions return promises as this ensures
that composite commands run in a deterministic and unsurprising order.

Also, making `:sanitize` synchronous might fix all issues we have with
Tridactyl not sourcing tridactylrcs entirely.

Note: I didn't even try to compile/test this because I don't want to do that on my computer. I'll amend my commits if CircleCI tells me I made a mistake.

### CODE REVIEW

File: src/excmds.ts [hunk 1]
```diff
@@ -175,8 +175,8 @@ ALL_EXCMDS = {
 
 /** @hidden **/
 //#background
-export async function getNativeVersion(): Promise<void> {
-    Native.getNativeMessengerVersion()
+export async function getNativeVersion(): Promise<number> {
```
Comment (bovine3dom) [inline src/excmds.ts]: Calling it a number seems dangerous, e.g. 0.19 > 0.2 should be true, and 0.19.1 is not a number

Comment (glacambre) [reply src/excmds.ts]: You're right, I just blindly copied what I found in https://github.com/tridactyl/tridactyl/blob/27572a59149ed7a3d423a531c1af664d82d20c1c/src/lib/native.ts#L81-L91 . I guess I'll update both :)

**Comment by bovine3dom** [comment]
```

ERROR in ./src/lib/native.ts
Module build failed (from ./node_modules/ts-loader/index.js):
Error: TypeScript emitted no output for /home/circleci/project/src/lib/native.ts.
    at makeSourceMapAndFinish (/home/circleci/project/node_modules/ts-loader/dist/index.js:80:15)
    at successLoader (/home/circleci/project/node_modules/ts-loader/dist/index.js:68:9)
    at Object.loader (/home/circleci/project/node_modules/ts-loader/dist/index.js:22:12)
 @ ./src/background.ts 18:0-42 34:4-10 84:12-22 171:0-15

ERROR in /home/circleci/project/src/lib/native.ts
./src/lib/native.ts
[tsl] ERROR in /home/circleci/project/src/lib/native.ts(91,9)
      TS2322: Type 'number' is not assignable to type 'string'.
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

Exited with code exit status 2
```

:)

**Comment by bovine3dom** [comment]
```

ERROR in ./src/lib/native.ts
Module build failed (from ./node_modules/ts-loader/index.js):
Error: TypeScript emitted no output for /home/circleci/project/src/lib/native.ts.
    at makeSourceMapAndFinish (/home/circleci/project/node_modules/ts-loader/dist/index.js:80:15)
    at successLoader (/home/circleci/project/node_modules/ts-loader/dist/index.js:68:9)
    at Object.loader (/home/circleci/project/node_modules/ts-loader/dist/index.js:22:12)
 @ ./src/background.ts 18:0-42 34:4-10 84:12-22 171:0-15

ERROR in /home/circleci/project/src/lib/native.ts
./src/lib/native.ts
[tsl] ERROR in /home/circleci/project/src/lib/native.ts(91,9)
      TS2322: Type 'number' is not assignable to type 'string'.
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

Exited with code exit status 2
```

:)

**Pull request merged**